services/QuillLMS/db/migrate/20210319160956_create_feedback_history_ratings.rb

```diff
@@ -1,7 +1,7 @@
 class CreateFeedbackHistoryRatings < ActiveRecord::Migration
   def change
     create_table :feedback_history_ratings do |t|
-      t.string :rating, null: false
+      t.boolean :rating
```

Comment:
I removed the null check - on the fence, but this keeps the find_or_create_by logic simpler

Reply:
I think, re: @anathomical's point on the model file, that you can keep the null check here, because we should be using the lack of record rather than the lack of rating to determine that a given user hasn't rated a given piece of feedback.